### DISCUSSION TITLE
Zero wallet

### DISCUSSION THREAD

**Discussion by zombie665** [Chia-Network/chia-blockchain#3699] [Support]
Please tell me WTF?
Proof on plots?
Where my money Lebovsky???
![8D586DE0-0853-4402-944D-E28DBD6E7C6C](https://user-images.githubusercontent.com/83660122/117036041-721c1800-ad1e-11eb-984f-d83646ad97f2.png)
![5B7A58CC-1005-4590-BA93-0F23C650A17E](https://user-images.githubusercontent.com/83660122/117036056-76e0cc00-ad1e-11eb-9b9f-a411c2e1bffa.png)

**Comment by dorkmo** [comment]
how many plots do you have? It can take days or months to win

**Reply by zombie665** [reply]
22 (17 K-33 and 5 K-32) plots )))

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
